index.json

```diff
@@ -1,4 +1,5 @@
 [
+  "",
```

Comment:
If you can remove this empty line I'll get this pulled in. Thanks!

Reply:
Sorry, fixed!